osu.Game/Overlays/Options/SliderOption.cs

```diff
@@ -104,13 +105,14 @@ public OsuSliderBar()
                         AutoSizeAxes = Axes.None,
                         RelativeSizeAxes = Axes.None,
                         Masking = true,
-                        BorderColour = new Color4(255, 102, 170, 255),
+                        BorderColour = OsuColor.SliderbarNub,
                         BorderThickness = 3,
                         Children = new[]
                         {
                             new Box
                             {
-                                Colour = new Color4(255, 102, 170, 0),
+                                Colour = new Color4(OsuColor.SliderbarNub.R,

```

Comment:
I generally prefer float over byte, and I think we should use it everywhere where colours are used, given that their internal representation is one float per channel. Also 0-1 makes more intuitive sense than 0-255. Let's see what @peppy 's opinion is about this first, though.